scep/certs_selector.go

```diff
@@ -0,0 +1,32 @@
+package scep
+
+import "crypto/x509"
+
+// CertsSelectorFunc is a type of function that filters certificates.
+type CertsSelectorFunc func([]*x509.Certificate) []*x509.Certificate
```

Comment:
Oh, sorry! Didn't mean to _get rid of_ the `CertsSelector` interface. Just that `CertsSelectorFunc` *satisfies* the `CertsSelector` interface. So folks can provide the implementation that they want.

Reply:
Nice one!!
Done!